include/boost/geometry/arithmetic/general_form.hpp

```diff
@@ -0,0 +1,161 @@
+// Boost.Geometry
+
+// Copyright (c) 2018-2019 Barend Gehrels, Amsterdam, the Netherlands.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_ARITHMETIC_GENERAL_FORM_HPP
+#define BOOST_GEOMETRY_ARITHMETIC_GENERAL_FORM_HPP
+
+#include <boost/geometry/util/math.hpp>
+#include <boost/geometry/core/access.hpp>
+#include <boost/geometry/core/config.hpp>
+#include <boost/geometry/util/select_most_precise.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace arithmetic
+{
+
+//--------------------------------------------------------------------------
+// Structure containing the General Form of a line, a*x + b*y + c == 0
+// Might be conceptized later. Therefore operations are implemented outside
+// the structure itself.
+template <typename GeneralType = double>
+struct general_form
+{
+    general_form()
+        : a(GeneralType())
+        , b(GeneralType())
+        , c(GeneralType())
```

Comment:
Why not `a(0)`? :) What is our decision about zero-initialization regarding new code? Because we agreed to leave the old code as it is but what about newly added one?

Reply:
Will update. I wrote this already for the recent discussion.